setup.py

```diff
@@ -71,6 +72,16 @@ def get_version(package):
     package_data={'integration/firmwares': ['integration/firmwares/*']},
     include_package_data=True,
     scripts=SCRIPTS,
+    entry_points={
+        'iotlab.commands': [
+            'admin=iotlabcli.parser.admin:main',
```

Comment:
From the setuptools documentation, IIUC you are messing up available command lines during installation. This line will install an `admin` command line. I think it should be iotlab-admin. Same for the lines below.

I also think that the `scripts` parameter could be removed, as well as all scripts in `bin/`

Reply:
Tested in a virtualenv, after a setup.py install there is an `iotlab-admin` (created by the scripts=... argument), but no `admin`

```
entry_points={
        'console_scripts': [
            'admin=iotlabcli.parser.admin:main',
```
would have added a console script, right now with 'iotlab.commands' this just registers an entry point without creating any script. I keep the scripts=... for backwards compatibility